autoPyTorch/pipeline/components/setup/lr_scheduler/CyclicLR.py

```diff
@@ -80,10 +80,11 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> BaseLRComponent:
         return self
 
     @staticmethod
-    def get_properties(dataset_properties: Optional[Dict[str, Any]] = None) -> Dict[str, str]:
+    def get_properties(dataset_properties: Optional[Dict[str, Any]] = None) -> Dict[str, Union[str, bool]]:
         return {
             'shortname': 'CyclicLR',
-            'name': 'Cyclic Learning Rate Scheduler',
+            'name': 'CyclicLR',
```

Comment:
Can we create a separated document for the naming rules for `shortname` and `name`?
I actually have not felt the necessity of `shortname` though.